XamlControlsGallery/ControlPages/ButtonPage.xaml

```diff
@@ -58,6 +58,45 @@
                 </x:String>
             </local:ControlExample.Xaml>
         </local:ControlExample>
+        <local:ControlExample HeaderText="Wrapping Buttons with large content">
+            <StackPanel MaxWidth="500">
```

Comment:
You don't need to specify this MaxWidth, either. It actually causes problems with large text:
![image](https://user-images.githubusercontent.com/13142665/97760854-6b87bc00-1ac1-11eb-89f0-3a9854643adc.png)

If you simply let the StackPanel manage its own width automatically, things look much better.

Reply:
Great suggestion, thank you! Fixed now.